server/src/models/finances/schema.js

```diff
@@ -1,9 +1,96 @@
+import { 
+  bilingual_field, 
+} from '../schema_utils';
 
 const schema = `
+  extend type Org{
+    org_vote_stat_pa: [OrgVoteStatPa]
+    org_vote_stat_qfr: [OrgVoteStatQfr]
+    org_vote_stat_estimates: [OrgVoteStatEstimates]
+    org_transfer_payments: [OrgTransferPayments]
+  }
+  
   extend type Program{
+    program_sobjs: [ProgramSobjs]
+    program_vote_stat: [ProgramVoteStat]
     program_spending: [ProgramSpending]
     program_fte: [ProgramFte]
   }
+
+  type OrgVoteStatPa{
+    dept_code: String
```

Comment:
Minor nitpick, but `dept_code` here's redundant since the `OrgVoteStatPa` type can only be reached through an `Org`, which already knows its own `dept_code`. Same with the `program_id` on `ProgramVoteStat` for example.